tests/supervisor/test_views.py

```diff
@@ -24,3 +25,16 @@ async def test_allocation_fails_on_invalid_item_hash(aiohttp_client):
             "type": "value_error.unknownhash",
         },
     ]
+
+
+@pytest.mark.asyncio
+async def test_system_usage(aiohttp_client):
+    """Test that the usage system endpoints responds. No auth needed"""
+    app = setup_webapp()
+    client = await aiohttp_client(app)
+    response: web.Response = await client.get("/about/usage/system")
+    assert response.status == 200
+    # check if it is valid json
+    resp = await response.json()
+    assert "cpu" in resp
+    assert resp["cpu"]["count"] > 0
```

Comment:
To make the test reliable on the future, maybe we can mock the system functions that returns the CPU specs and test here the options to have a match of the same results.